eternalcore-core/src/main/java/com/eternalcode/core/feature/randomteleport/RandomTeleportHeightRangeRepresenter.java

```diff
@@ -0,0 +1,41 @@
+package com.eternalcode.core.feature.randomteleport;
+
+import net.dzikoysk.cdn.entity.Contextual;
+import net.dzikoysk.cdn.entity.Description;
+
+@Contextual
+public class RandomTeleportHeightRangeRepresenter {
```

Comment:
package-private and RandomTeleportHeightRange?

Reply:
no, because cdn is crying

Reply:
Make this for api, we don't need clear impl of configuration you can map it or something